agent/handlers/task_server_setup_test.go

```diff
@@ -1101,11 +1128,11 @@ func TestTaskHTTPEndpointErrorCode400(t *testing.T) {
 		"/v3/wrong-v3-endpoint-id",
 		"/v3/",
 		"/v3/wrong-v3-endpoint-id/stats",
-		"/v3//stats",
+		"/v3/stats",
```

Comment:
Why was this changed? Shouldn't `"/v3//stats"` work as well?

Reply:
we handle double/muti slash with a redirect prior to 1.20.3 release. at 1.21.0 release we changed the router and added the behavior to not clean double/multi slash. This is the root cause of this issue https://github.com/aws/amazon-ecs-agent/issues/1839.